packages/superset-ui-connection/src/SupersetClient.ts

```diff
@@ -34,6 +35,7 @@ const SupersetClient: SupersetClientInterface = {
   isAuthenticated: () => getInstance(singletonClient).isAuthenticated(),
   post: (request: RequestConfig) => getInstance(singletonClient).post(request),
   reAuthenticate: () => getInstance(singletonClient).init(/* force = */ true),
+  request: (request: RequestConfig) => getInstance(singletonClient).get(request),
```

Comment:
I don't think we should expose a general `request` method on the public interface, there are only 4 possible methods so if you want `delete` or `put` I would just add those.

Reply:
+1

Reply:
Looks like there are many more methods: https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods

Happy to add more explicit `delete` and `put` as they are most common. Can I leave `request` just in case? If someone has an edge case where they need to use one of the more obscure methods, it sucks to have to come and add it here...